### PR TITLE
CI/CD updates

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-4654)

## Description
Change the CI/CD installation to "upload" the packs instead of install it to avoid the force install dependencies:
- Added the **install_dependencies** argument to **ContentPackInstaller** script.
- Use the **ContentPackInstaller** script instead of the **MarketplacePackInstaller** script in **Configuration Setup** playbook.
- Added the **demisto-api-install-packs** command to **Demisto REST API** integration - the command download the pack zip from the marketplace bucket and upload it to the server.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js [hunk 1]
```diff
@@ -152,6 +154,55 @@ var deleteIncidents = function(ids_to_delete, fields_to_keep) {
...
+    });
+
+    if (res.StatusCode < 200 || res.StatusCode >= 300) {
+        throw 'Demisto REST APIs - Failed to download pack file from ' + pack_url;
+    }
+
+    let file_path = res.Path
+
+    // upload the pack
+    sendMultipart('contentpacks/installed/upload?skipVerify=true&skipValidation=true', file_path,'{}');
```
Comment (DeanArbel) [inline Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js:174]: `skipVerify=true&skipValidation=true` can make this command a bit risky. It's ok to skip verification/validation, but let's add these as default arguments, so it'll be better reflected to the user.

File: Packs/ContentInstallation/Scripts/ContentPackInstaller/ContentPackInstaller.py [hunk 2]
```diff
@@ -152,22 +156,26 @@ def install_packs(self, packs_to_install: List[Dict[str, str]]) -> None:
...
+                {
+                    'packs_to_install': str(pack_payload)
+                },
+                fail_on_error=False,
+            )
+
+            if not status:
+                demisto.error(f'{SCRIPT_NAME} - Failed to install the pack {pack["id"]} - {str(res)}')
+                self.packs_failed[pack['id']] = str(pack['version'])
+                return
```
Comment (DeanArbel) [inline Packs/ContentInstallation/Scripts/ContentPackInstaller/ContentPackInstaller.py:177]: Why not continue with the other packs that didn't fail installation?

**Review by DeanArbel — CHANGES_REQUESTED** [review]
Good job, just a couple of notes.
Please make sure to have a technical writer review this as well.

File: Packs/ContentInstallation/Scripts/ContentPackInstaller/ContentPackInstaller.py [hunk 3]
```diff
@@ -36,6 +37,9 @@ def get_installed_packs(self) -> None:
             demisto.debug(error_message)
             return
 
+        if type(res) is list:
+            res = res[0]
```
[the remark this answers is not in the document]
Comment (adi88d) [reply Packs/ContentInstallation/Scripts/ContentPackInstaller/ContentPackInstaller.py:42]: done

File: Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js (hunk 1 above)
Comment (adi88d) [reply Packs/DemistoRESTAPI/Integrations/DemistoRESTAPI/DemistoRESTAPI.js]: added as arguments

**Review by DeanArbel — APPROVED** [review]
Good job!

**Pull request merged**